### PR TITLE
add brain-tumor-segmentation-0002 to OMZ

### PR DESCRIPTION
**Pull request by elenash**

### CODE REVIEW

**Comment by elenash** [comment]
@IRDonch Please take a look. What's more need to update (probably some docs about onnx framework type)?

**Comment by vladimir-dudnik** [comment]
@elenash Does this models required OpenVINO 2019R4 to convert to IR?

**Comment by elenash** [comment]
I checked this model with build openvino_2019.4.469 and current master ( commit b2d5a38c7fcfb2c4fc8b8470215c82642817631a). It converts successfully.

**Comment by elenash** [comment]
> @elenash Does this models required OpenVINO 2019R4 to convert to IR?

Yes. Does precommit still use R3?

**Comment by vladimir-dudnik** [comment]
@elenash yes, we still not have stable enough R4 build. Ok, we will re-test and merge when pre-release R4 build will be available

File: tools/downloader/common.py [hunk 1]
```diff
@@ -36,6 +36,7 @@
     'mxnet': None,
     'pytorch': 'pytorch_to_onnx.py',
     'tf': None,
+    'onnx': None
```
Comment (IRDonch) [inline tools/downloader/common.py]: Please keep the list in alphabetical order. Also, update the downloader readme and `CONTRIBUTING.md`.

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Brain-tumor-segmentation-0002
+
+## Use Case and High-Level Description
+
+This model was created for participation in the [Brain Tumor Segmentation Challenge](https://www.med.upenn.edu/cbica/brats2019/registration.html) (BraTS) 2019. The code is based on the corresponding paper[TBD](), where the authors employ knowledge distillation for automatic brain tumor segmentation. First, they annotated unlabeled data with ensemble of different models. Then, they trained simple UNet model with residual blocks.
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Presumably you're going to fill in the paper later?

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+With the following channels: `whole tumor`, `tumor core` and `enhancing tumor`.
+
+## Legal Information
+
+[LICENSE](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE)
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Please update the license description according to the new convention (you can use `brain-tumor-segmentation-0001` as an example).

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md (hunk 2 above)
Comment (elenash) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Right

Comment (IRDonch) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: Then leave the conversation open until you do. :-)

File: tools/downloader/common.py [hunk 4]
```diff
@@ -34,8 +34,9 @@
     'caffe2': 'caffe2_to_onnx.py',
     'dldt': None,
     'mxnet': None,
+    'onnx': None,
     'pytorch': 'pytorch_to_onnx.py',
-    'tf': None,
+    'tf': None
```
Comment (IRDonch) [inline tools/downloader/common.py]: Please don't do this.

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+With the following channels: `whole tumor`, `tumor core` and `enhancing tumor`.
+
+## Legal Information
+
+The original model is distributed under the
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]:
```suggestion
The original model is distributed under the following
```

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+[license](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE).
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]:
```suggestion
[license](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE):
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -9,6 +9,8 @@ Frameworks supported by the Open Model Zoo:
 * PyTorch\* (via conversion to ONNX\*)
 * MXNet\*
 
+Open Model Zoo also supports models being converted to ONNX format.
```
Comment (IRDonch) [inline CONTRIBUTING.md]: This is just redundant with what is already written above. ITYM "models already in the ONNX format".

Comment (elenash) [reply CONTRIBUTING.md]: Ok, so I won't update CONTRIBUTING.md any how?

File: tools/downloader/common.py (hunk 4 above)
Comment (elenash) [reply tools/downloader/common.py]: Ok, will get back

File: models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md (hunk 2 above)
Comment (elenash) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: We need to add this model in R4 while the paper can be submitted later. I'd propose to merge with such message or I can remove any notes about the article for the moment

Comment (vladimir-dudnik) [reply models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md]: I think it should be enough to say that model was created to participate challenge and it have unet arch trained on residual blocks.

File: CONTRIBUTING.md (hunk 7 above)
Comment (IRDonch) [reply CONTRIBUTING.md]: Well, no, you still need to mention that ONNX models are supported. I suggested you a possible wording.

Comment (elenash) [reply CONTRIBUTING.md]: Ok!

**Comment by elenash** [comment]
@IRDonch Can it be merged? This model requires MO r4 to be converted to IR, so precommit is red.

**Comment by IRDonch** [comment]
>  Can it be merged? This model requires MO r4 to be converted to IR, so precommit is red.

I think you answered your own question. :-) We can merge it after I update CI with a pre-release OpenVINO build.

File: models/public/brain-tumor-segmentation-0002/model.yml [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+# limitations under the License.
+
+description: >-
+  This model was created for participation in the Brain Tumor Segmentation Challenge
+  <https://www.med.upenn.edu/cbica/brats2019/registration.html> (BraTS) 2019.
+  It has UNet architecture trained with residual blocks.
+task_type: semantic_segmentation
+files:
+  - name: brain-tumor-segmentation-0002.onnx
+    size: 21225
```
Comment (IRDonch) [inline models/public/brain-tumor-segmentation-0002/model.yml]: This size is obviously incorrect. The file is many megabytes in size.

Comment (elenash) [reply models/public/brain-tumor-segmentation-0002/model.yml]: Ok, I'll fix it. why there is no any check on downloaded size?

Comment (elenash) [reply models/public/brain-tumor-segmentation-0002/model.yml]: Fixed

Comment (IRDonch) [reply models/public/brain-tumor-segmentation-0002/model.yml]: > why there is no any check on downloaded size?

There is now. :-)

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Pull request merged**